spaces/S000086/properties/P000104.md

```diff
@@ -0,0 +1,16 @@
+---
+space: S000086
+property: P000104
+value: false
+refs:
+- mathse: 310113
+  name: "Accumulation points of uncountable sets"
+---
+
+If $d:X^2\to[0,+\infty)$ was a symmetric, then for every $x\in\mathbb R$ there would exist $r_x>0$ such that $B_d((x,0),r_x)\subseteq \mathbb R\times\{0\}$.
+Let $A_n:=\{x\in\mathbb R: r_x>1/n\}$.
+Since $\mathbb R = \bigcup_{n=1}^\infty A_n$ is uncountable, so is $A_m$ for some $m\geq 1$.
+
+An uncountable subset of $\mathbb R$ has an accumulation point (cf. {{mathse:310113}}), hence there exists a sequence of distinct values $(a_n)_{n=1}^\infty\subset A_m$ such that $a_n\to a\in \mathbb R$.
+Let $S:=\overline{\{(a_n,0):n\in\mathbb N\}}=\{(a_n,0):n\in\mathbb N\}\cup\{(a,0),(a,1)\}$ and $S_0:=S\setminus\{(a,1)\}$.
+By closedness of $S$, every point $u\in X\setminus S$ has a positive distance $d$ to $S_0\subset S$. By definition of $A_m$, we have $d((a_n,0),(a,1))\geq 1/m$, hence the point $(a,1)$ has also positive distance to $S_0$ but $S_0$ is not closed in $X$.
```

Comment:
```suggestion
If $d:X^2\to[0,+\infty)$ was a symmetric generating the topology, then for every $x\in\mathbb R$ there would exist $r_x>0$ such that $B_d((x,0),r_x)\subseteq \mathbb R\times\{0\}$.
Let $A_n:=\{x\in\mathbb R: r_x>1/n\}$.
Since $\mathbb R = \bigcup_{n=1}^\infty A_n$ is uncountable, so is $A_m$ for some $m\geq 1$.

An uncountable subset of $\mathbb R$ has an accumulation point (cf. {{mathse:310113}}), hence there exists a sequence of distinct values $(a_n)_{n=1}^\infty\subset A_m$ such that $a_n\to a\in \mathbb R$.
Let $S:=\overline{\{(a_n,0):n\in\mathbb N\}}=\{(a_n,0):n\in\mathbb N\}\cup\{(a,0),(a,1)\}$ and $S_0:=S\setminus\{(a,1)\}$.
Since $S$ is closed, every point $u\in X\setminus S$ satisfies $d(u,S_0)\ge d(u,S)>0$.
By definition of $A_m$, we also have $d((a_n,0),(a,1))\geq 1/m$, hence hence $d((a,1),S_0)>0$.
Since every point not in $S_0$ has a positive distance to $S_0$, $S_0$ is closed, which is a contradiction.
```